recipes/r-sigclust/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = '1.1.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sigclust
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: sigclust_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/sigclust_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/sigclust/sigclust_{{ version }}.tar.gz
+  sha256: efc4d8b8a19208efee9e86bbfd1aec09cac4fe3340033992f59e5892e9d9a104
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  skip: True  # [win]
+
+requirements:
+  build:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('sigclust')"  # [not win]
+    - "\"%R%\" -e \"library('sigclust')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=sigclust
+  license: GPL (>= 2)
+  summary: SigClust is a statistical method for testing the significance of clustering results.
+    SigClust can be applied to assess the statistical significance of splitting a data
+    set into two clusters. For more than two clusters, SigClust can be used iteratively.
+  license_family: GPL3
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3'  # [win]
+
+extra:
+  recipe-maintainers:
+    - cbrueffer
+    - johanneskoester
```

Comment:
@johanneskoester 
@bgruening
@daler
@jdblischak
are you ok with being added as maintainers for this recipe?

Reply:
@CJ-Wright Yes, our setup is to add the 4 of us as maintainers for the conda-forge R packages. This makes it easier for us to make sure we can keep them all up to date.

https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml